eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -226,8 +231,13 @@ jobs:
 
   - ${{ if eq(parameters.targetOS, 'windows') }}:
     - script: |
-        call $(sourcesPath)\build.cmd -ci -cleanWhileBuilding -prepareMachine /p:TargetOS=${{ parameters.targetOS }} /p:TargetArchitecture=${{ parameters.targetArchitecture }} ${{ parameters.extraProperties }}
+        call $(sourcesPath)\build.cmd -ci -cleanWhileBuilding -prepareMachine %devArgument% /p:TargetOS=${{ parameters.targetOS }} /p:TargetArchitecture=${{ parameters.targetArchitecture }} ${{ parameters.extraProperties }}
       displayName: Build
+      env:
+        ${{ if eq(parameters.useDevVersions, 'True') }}:
+          devArgument: -dev
+        ${{ else }}:
+          devArgument: ''
```

Comment:
Why do we need to set this as an env var?

Reply:
It's easier to set it as an env var than to write the conditional into a batch script. It's not required, but I think it's easier to understand than the alternatives.

Reply:
What I don't like about it is that it flows into msbuild and is available as a property because of the automated env var -> property translation.

Reply:
Yeah I'm not enthused about that, but I do think this is much more understandable than trying to do what we do above for bash with batch.

Reply:
Right. I think we have a tracking issue to just use YML variables instead of defining the build command in bash or cmd.